AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -171,6 +171,9 @@ modepark.de#$#body { position: static!important; overflow: visible!important; }
 msmode.de##body .cookie-warning
 ouniversodatv.com###litespot-pro-cookie-ify
 trudvsem.ru##.cookies
+jobijoba.at#$##dialog { display: none !important; }
+jobijoba.at#$#.modal-backdrop { display: none !important; }
+jobijoba.at#$#body.modal-open { overflow: auto !important; }
```

Comment:
This is a bit of a wide solution. The safest option here is to add a JS rule - fixed it directly.